messaging/ses/src/sendEmail.ts

```diff
@@ -0,0 +1,82 @@
+import * as fetch from 'isomorphic-fetch';
+import * as FormData from 'form-data';
+import * as aws from 'aws-sdk';
+import { FunctionEvent } from 'graphcool-lib';
+
+const ses = new aws.SES({
+  region: process.env.AWS_REGION,
+  accessKeyId: 'AWS_ACCESS_KEY_ID',
+  secretAccessKey: 'AWS_SECRET_ACCESS_KEY',
+});
+
+
+interface EventData {
+  from: string
+  to: [string]
+  subject: string
+  text: string
+  html: string
+  recipientVariables: Object | null
+}
+
+
+
+
+export default async (event: FunctionEvent<EventData>) => {
+  console.log(event)
+  console.log('ENV:', process.env) 
+
+  if (!process.env['ACCESS_KEY_ID']) {
+    console.log('Please provide a valid AWS Access Key ID!');
+    return { error: 'Module not configured correctly.' }
+  }
+
+  if (!process.env['SECRET_ACCESS_KEY']) {
+    console.log('Please provide a valid AWS Secret Access Key!');
+    return { error: 'Module not configured correctly.' }
+  }
+
+  try {
+
+    const { to, from, subject, text, html } = event.data
+    const recipientVariables = event.data.recipientVariables || {}
+
+    const params = {
+      Destination: {
+        ToAddresses: ['johndoe@gmail.com']
+      },
+      Message: {
+        Body: {
+          Html: {
+            Charset: 'UTF-8',
+            Data: html
+          },
+          Text: {
+            Charset: 'UTF-8',
+            Data: text
+          }
+        },
+        Subject: {
+          Charset: 'UTF-8',
+          Data: subject
+        }
+      },
+      ReturnPath: from,
+      Source: from
+    }
+
+    ses.sendEmail(params, (err, data) => {
```

Comment:
callbacks don't reliably work. wrap this in a Promise instead. Here's an example from a different context

```js
  return new Promise((resolve, reject) => {
    stripe.charges.create(
      {
        amount,
        description,
        currency: 'usd',
        customer: stripeCustomerId,
      },
      (err, charge) => {
        if (err) {
          console.log(`Error creating Stripe charge: ${JSON.stringify(err)}`)
          reject(err)
        } else {
          console.log(`Successfully created Stripe charge: ${charge.id}`)
          resolve(charge.id)
        }
      },
    )
  })
```